crates/re_viewer/data/quick_start_guides/cpp_connect.md

```diff
@@ -1,3 +1,45 @@
 ## C++ Quick Start
 
-TODO(ab): https://github.com/rerun-io/rerun/issues/3870
+${SAFARI_WARNING}
+
+#### Installing the Rerun viewer
+The Rerun C++ SDK works by connecting to an awaiting Rerun Viewer over TCP.
+
+Building from source:
+* [Install cargo](https://rustup.rs/)
+* `cargo install rerun-cli`
+
+Using `pip`:
+* `pip install rerun-sdk`
+
+After you've installed it, type `rerun` in your terminal to start the viewer.
+
+
+#### Using the Rerun C++ SDK
+First install the `arrow-cpp` library on your system using your favorite package manager.
+
+Then add the following to your `CMakeLists.txt`:
+
+```cmake
+include(FetchContent)
+FetchContent_Declare(rerun_sdk URL https://build.rerun.io/commit/06dd483/rerun_cpp_sdk.zip) # 2023-10-20
```

Comment:
```suggestion
FetchContent_Declare(rerun_sdk URL https://github.com/rerun-io/rerun/latest/download/rerun_cpp_sdk.zip)
```

Reply:
404

Reply:
Yeah it will 404 until we have a cpp release

Reply:
I've temporarily added the C++ sdk as an asset of the 0.9.1 release so we can test our links.

Reply:
'latest' is actually a bad idea for a users's cmake script. Because then it auto-updates when we bring out a new latest

Reply:
`https://github.com/rerun-io/rerun/releases/download/0.10.0/rerun_cpp_sdk.zip` then